### PR TITLE
Single program

### PR DESCRIPTION
**Pull request by arunsridher-soal**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (ssakethraju) [inline package-lock.json]: Not required on the root folder

File: package.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (ssakethraju) [inline package.json]: not required on the root folder

File: single-program/.env.example [hunk 3]
```diff
@@ -0,0 +1,8 @@
+CONTRACT_ID = 
+NEAR_ENV = 
+NODE_ENV = 
+ACCOUNT_ID = 
+METHOD_NAME = 
+ATTACHED_DEPOSIT = 
+PRIVATE_KEY = 
+MAX_GAS = "300000000000000"
```
Comment (ssakethraju) [inline single-program/.env.example]: Why `MAX_GAS` is required in their project env?

File: single-program/near-interface.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+export default async function giveReward(userAccountId) {
+  const args = {
+    reward_name: "video_likes",
+    program_name: "pe",
+    user_wallet: userAccountId,
+    program_owner: process.env.ACCOUNT_ID,
+    private_key: process.env.PRIVATE_KEY,
+  };
+  const reward = await give_rewards(
```
Comment (ssakethraju) [inline single-program/near-interface.js:24]: Can we display status message without querying the transaction (  getTransactionDetails ).

**Review by ssakethraju — CHANGES_REQUESTED** [review]

File: single-program/.env.example (hunk 3 above)
Comment (arunsridher-soal) [reply single-program/.env.example]: Last I checked it is needed in the x2earn sdk

File: single-program/near-interface.js (hunk 4 above)
Comment (arunsridher-soal) [reply single-program/near-interface.js:24]: Will check with Vaulstein

File: single-program/.env.example (hunk 3 above)
Comment (ssakethraju) [reply single-program/.env.example]: If it is required in x2earn sdk, this variable should be directly in the contract code it self but here. Please check with vaulstein

Comment (arunsridher-soal) [reply single-program/.env.example]: ok

**Review by ssakethraju — APPROVED** [review]

**Pull request merged**